source/projects/medienkompetenzen/de/index.md

```diff
@@ -0,0 +1,100 @@
+
+## Pilotprojekt
+
+**Die Zentral- und Landesbibliothek Berlin (ZLB)** und die **Technologiestiftung Berlin** entwickeln gemeinsam ein Pilotprojekt mit dem Ziel, Bürger\*innen Grundwissen zu Medien- und Digitalkompetenzen zu vermitteln, damit Bürger\*innen die zunehmenden Serviceangebote staatlicher Dienstleistungen sicher und selbständig erledigen können und dabei eine vollwertige gesellschaftliche Teilhabe, auch dort, wo sich diese auf digitale Prozesse stützt, erlangen können.
+
+## Planung und Organisation
+
+Drei Bibliotheken in Berlin (Buch, Berlin/Mitte und Neukölln) werden das Pilotprojekt starten. An jedem dieser Standorte wird ein voll ausgestatteter und geschützter digitaler Arbeitsplatz, mit Beratungsleistung durch Mitarbeiter\*innen der Bibliothek zur Verfügung stehen. Die betreuenden Bibliotheksmitarbeiter\*innen erhalten im Projektrahmen eine gezielte Schulung. Nutzer\*innen bekommen so Hilfe, um zu erlernen, wie sie online Verwaltungsleistungen wie z.B.: Beantragung eines Führungszeugnisses, Kita-Gutscheine, Terminanfragen beim Bürgeramt etc. anfordern und bearbeiten können. 
+
+Es wird keine Fach- oder Rechtsberatung angeboten, sondern Beratung und Unterstützung beim Erlernen von Fähigkeiten, die zur Nutzung des digitalen Verwaltungsangebotes grundlegend nötig sind, wie z.B. das Suchen und Finden von Informationen. 
+
+## Das Pilotprojekt soll herausfinden
+
+- welche Unterstützung die Nutzer\*innen bei Online-Prozessen brauchen
+- welche bedarfsorientierten Programmangebote für die Vermittlung digitaler Fähigkeiten gebraucht werden
+- welche Online-Dienstleistungen für Bürger\*innen es gibt und wie deren Nutzbarkeit aussieht
+- wie Serviceangebote weiterentwickelt werden können
+- wie digitale Souveränität verbreitet werden kann
+- welcher Beitrag im Projektrahmen zur Digitalen Daseinsvorsorge geleistet werden kann
+
+{% include macro-image-section-markdown.html src="/projects/medienkompetenzen/images/dienstleistungen.jpg" caption="In Workshops fragten wir Teilnehmer\*innen: Welche staatlichen Online-Dienstleistungen wünschen Sie sich? Und welche Betreuung benötigen Sie dabei?" %}
+
+## Online-Dienstleistungen
+
+In der ersten Phase des Pilotprojektes wird an allen 3 Standorten ein Angebot für folgende Verwaltungsleistungen zur Verfügung stehen:
+
+**Services der Bürgerämter**
+* Bürgeramt Anmeldung (Termine)
+* eID (elektronische Identifikation) Anmeldung und Nutzungsmöglichkeiten
+* Anmeldung KITA / KITA Gutschein
+* Führungszeugnis
+* Wohnung An- und Ummeldung
+
+**Leistungen für Migranti*nnen**
+* Aufenthaltsgenehmigung
+* Arbeitsgenehmigung für Student\*innen
+
+**Organisatorische Dienstleistungen**
+* E-Mail Einrichten
+* Arzttermine online machen
+
+![Digital- Medienkompetenzen in der Bibliothek](/projects/medienkompetenzen/images/thumbnail.jpg)
```

Comment:
I think this should not be here.

Reply:
@edgalindo You have the markdown img link and the include

Reply:
if I use following {% include macro-image-section-markdown.html src="/projects/medienkompetenzen/images/thumbnail.jpg" caption="Digital- Medienkompetenzen in der Bibliothek" %}, the image size is too big for the area

Reply:
Means the image is to small and gets pumped up to fit the column I guess.
 You meved back to having it included using `![alt text](path/to/img.ext)`

One thing you _could_ do is embed it into a non rectangular frame with a size larger then the column.